### ISSUE TITLE
Any chance of .jpz support?

### ISSUE THREAD

**Issue by jseakle** [ben4808/crosswords-ui#5]
It's very useful for things as basic as including nonascii characters in clues, and many other things, but so few tools support it :/

**Comment by bzoon** [comment]
I agree that the feature set of .puz leaves a lot to be desired. This is a great idea, but I won't be implementing it any time soon, as I'm shifting focus to wordlist tools for the foreseeable future.
